### ISSUE TITLE
Watch Log for Tail log Files return too many emails

### ISSUE THREAD

**Issue by diglin** [magento-hackathon/Hackathon_MageMonitoring#40]
If the watch log is enabled to get tails of log files, too many emails are sent to the recipient even it the log files didn't change in the meantime.

Class to check and method: Hackathon_MageMonitoring_Model_Widget_Log_Tail::watch

**Comment by softint2** [comment]
Problem here too its set to 5 minutes, every 5 minute i get 5 emails(all same content) and it keeps going, even the log didnt change

**Comment by michaelscheel** [comment]
Vielen Dank für Ihre Nachricht. Ich bin bis einschließlich 08.01.2016, nicht im Büro. Ihre Nachricht wird nicht weitergeleitet. Ich werde diese nach meiner Rückkehr beantworten.

In dringenden Fällen wenden Sie sich bitte an meinen Kollegen Herrn Völker (voelker@virtualways.de), oder telefonisch unter der Rufnummer 0731 - 388 628 10.

Mit freundlichen Grüßen
Michael Scheel

VIRTUALWAYS GmbH
Geschäftsführer

T   0731 - 379 357 91
M   0151 - 405 108 45
W   http://www.virtualways.de
@   scheel@virtualways.de

**Comment by diglin** [comment]
@softint2 you can solve it by changing the configuration of the watchdog widget and disabling the email. There is a checkbox when you click on the mechanical wheel in the tab Watchdog. I don't have screenshot under the hand but it's not so difficult to find it with some cleverness.

**Comment by sprankhub** [comment]
Yeah, it is possible to disable it there, but it would still be much better if a mail was only sent if something changes.
